icons/laptop.svg

```diff
@@ -9,5 +9,6 @@
   stroke-linecap="round"
   stroke-linejoin="round"
 >
-  <path d="M20 16V7a2 2 0 0 0-2-2H6a2 2 0 0 0-2 2v9m16 0H4m16 0 1.28 2.55a1 1 0 0 1-.9 1.45H3.62a1 1 0 0 1-.9-1.45L4 16" />
+  <path d="M6 4a2 2 0 0 0-2 2v9h16V6a2 2 0 0 0-2-2Z" />
+  <path d="m20 15 1.3 3.1c.4 1-.2 1.9-1.3 1.9H4c-1.1 0-1.7-.8-1.3-1.9L4 15" />
```

Comment:
Definitely much better than it was, but I think the bottom part could be even wider:

```suggestion
  <path d="M20 6v9H4V6c0-1.1.9-2 2-2h12a2 2 0 0 1 2 2Z" />
  <path d="m20 15 1.81 3.02A1.3 1.3 0 0 1 20.69 20H3.3a1.31 1.31 0 0 1-1.12-1.98L4 15" />
```